### ISSUE TITLE
Format specification to represent ALL possible metadata fields

### ISSUE THREAD

**Issue by sandreas** [sandreas/tone#13]
Think over a metadata format (JSON?) that contains all possible metadata fields (`JsonMeta`) and a container, to wrap this format to contain even more information (`file`, `options`, etc.). Additionaly, a parameter `--json-path` should be implemented, where this output can be queried in `dump` command ([see here](https://www.newtonsoft.com/json/help/html/QueryJsonSelectTokenJsonPath.htm)), e.g.:

*Meta*:
```json
{
    "artist": "My Artist",
    "lyrics": {
          
     }
    "chapters": [ 
        [0, 10000, "chapter title"]
    ],
    "pictures": [{
           "id" : "..",

     }]
}
```

*Container:*
```json
{
 "options": {
    "meta-album": "My Album"
  },  
 "file": {
     "size": 800001,
     "path": "/audiobooks",
     "name": "my-audiobook.m4b"
     "created": "2020-08-20T12:34:56Z"
  }
  "meta": {"meta from above"}
}
```

# References:
*Chapters* - for now `start`, `length` and `title` would be enough
```c#
// start
// end / length
// title
// uniqueID (id3 only)
// subtitle (id3 only)
// url (id3 only)
// picture (id3 only)
```

*Pictures* - for now `data`, `picType` and `tagType` are enough (`position` can be calculated automatically?)
```c#
public static PictureInfo fromBinaryData(byte[] data, PIC_TYPE picType = PIC_TYPE.Generic, TagType tagType = TagType.ANY, object nativePicCode = null, int position = 1) {
}
```

**Comment by sandreas** [comment]
Introduced with `v0.0.6` (`tone.json`).

Main key `meta` has been implemented.
Missing is a main key for the audio and file specific information, e.g.:

```json
{
  "meta": {},
  "audio": {
    "codec": "aac",
    "duration": 834833,
    "format": "mp4",
    "bitrate": 64000
    "samplerate": 44100
  },
  "file": {
    "size": "13255",
    "created": "2020-01-01T12:34:56Z",
    "mimetype": "audio/x-m4b"
  }
}
```

**Comment by sandreas** [comment]
fixed with 0.0.8
